pywemo/subscribe.py

```diff
@@ -241,8 +251,16 @@ class HTTPServer(ThreadingHTTPServer):
 
 def _start_server() -> HTTPServer | None:
     """Find a valid open port and start the HTTP server."""
-    for i in range(0, 128):
-        port = 8989 + i
+    requested_port = os.getenv('PYWEMO_HTTP_SERVER_PORT')
+    if requested_port is not None and requested_port.isdigit():
+        start_port = int(requested_port)
+        ports_to_check = 1
+    else:
+        start_port = 8989
+        ports_to_check = 128
+
+    for i in range(0, ports_to_check):
```

Comment:
It was unlikely for this to fail before. Under the new logic there could be only 1 port that is attempted. Let's modify this a bit to give feedback in the form of a raised exception.

```python
    for i in range(0, ports_to_check):
        port = start_port + i
        try:
            return HTTPServer(('', port), RequestHandler)
        except OSError as error:
            last_error = error
    raise last_error
```